jaseci_core/jaseci/jac/interpreter/interp.py

```diff
@@ -1663,7 +1647,6 @@ def run_filter_compare(self, jac_ast, obj):
             return self.pop().value
         else:
             self.rt_error(f"{name} not present in object", kid[0])
-            return False
```

Comment:
Unreachable code.
Forcing error on filtering let the developer know that there are still elements that are not yet synced on the latest structure and they still need to do a proper migration